src/lib/isEAN.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * The most commonly used EAN standard is
+ * the thirteen-digit EAN-13, while the
+ * less commonly used 8-digit EAN-8 barcode was
+ * introduced for use on small packages.
+ * EAN consists of:
+ * GS1 prefix, manufacturer code, product code and check digit
+ * Reference: https://en.wikipedia.org/wiki/International_Article_Number
+ */
+
+import assertString from './util/assertString';
+
+/**
+ * Define EAN Lenghts; 8 for EAN-8; 13 for EAN-13
+ * and Regular Expression for valid EANs (EAN-8, EAN-13),
+ * with exact numberic matching of 8 or 13 digits [0-9]
+ */
+const LENGTH_EAN_8 = 8;
+const validEanRegex = /(?<!\d)(\d{8}|\d{13})(?!\d)$/;
```

Comment:
RegExp lookbehind assertions are an ES2018 feature. They are not supported in all platforms.
By the way, can't a much simpler regex like ```/^(\d{8}|\d{13})$/``` do the trick?

Reply:
@tux-tn Yeah you're right, the `/^(\d{8}|\d{13})$/` is sufficient to detect a consecutive sequence of 8-digits or 13-digits **only** ... Forgot about the lookbehind assertions being an ES2018 feature. Thanks! I've just modified it 👍